doc/about/roadmap.rst

```diff
@@ -0,0 +1,157 @@
+.. _roadmap:
+
+Roadmap
+=======
+
+This page contains the current roadmap for the Vega-Altair project. The roadmap is
+informed by the needs of the Vega-Altair community, and the priorities of the active
+project contributors. It's designed to communicate the direction of the project,
+but it's not a commitment that these items will be completed in a particular timeframe.
+If you would like to help contribute to any of these areas, or suggest new ones,
+please `start a discussion <https://github.com/altair-viz/altair/discussions>`_.
+
+Vega-Altair is deeply integrated with other components in the Vega ecosystem,
+and as such many items on the roadmap will require work in other projects.
+Abbreviations for these projects are included at the end of project descriptions
+where relevant:
+
+* *(VL)* — `Vega-Lite <https://vega.github.io/vega-lite/>`_
+* *(VG)* — `Vega <https://vega.github.io/vega/>`_
+* *(VF)* — `VegaFusion <https://vegafusion.io/>`_
+* *(VC)* — `VlConvert <https://github.com/vega/vl-convert>`_
+
+API Ergonomics
+--------------
+The primary job of the Vega-Altair library is to provide an ergonomic Python
+API for generating Vega-Lite JSON specifications, and there are several improvements
+here that we would like to investigate.
+
+Areas of focus:
+
+* Improve the syntax for creating condition expressions with two or more
+  predicates.
+
+* Explore the possibility of a new operator, ``*``, for modular compositing of sub components.
+  See this composition proposal
+  `queryverse/VegaLite.jl#411 <https://github.com/queryverse/VegaLite.jl/pull/411>`_ and
+  also https://aog.makie.org/stable/generated/penguins/#Tutorial.
+
+* Standardize the API of methods that convert charts into other formats (``alt.Chart().to_<format>``).
+
```

Comment:
```suggestion
* Standardize the API of methods that convert charts into other formats (``alt.Chart().to_<format>``).

* Add type hints to the public API and most of the internals so that users can type check their Altair code with a static type checker such as mypy. This will also make it easier for other packages to integrate with Altair.
```